backend/googlecloudstorage/googlecloudstorage.go

```diff
@@ -51,8 +51,7 @@ import (
 const (
 	rcloneClientID              = "202264815644.apps.googleusercontent.com"
 	rcloneEncryptedClientSecret = "Uj7C9jGfb9gmeaV70Lh058cNkWvepr-Es9sBm0zdgil7JaOWF1VySw"
-	timeFormatIn                = time.RFC3339
-	timeFormatOut               = time.RFC3339Nano
```

Comment:
> This will lose the precision from the timestamp won't it?

I'm not sure I understand what you mean. `timeFormatIn` was only used in `time.Parse(timeFormatIn, t)`, which gives the same result if you replace it with `time.RFC3339Nano`. `timeFormatOut` was assigned the same value as `time.RFC3339Nano`. And it was only used in `time.Format(timeFormatOut)`.

Reply:
Sorry I got confused and I thought you were replacing the format with `time.RFC3339` not `time.RFC3339Nano`.

I think this change is OK!